app/views/content/is-teaching-right-for-me/get-school-experience.md

```diff
@@ -23,6 +22,11 @@ promo_content:
 navigation: 5.50
 navigation_title: How do I get experience in a school?
 navigation_description: Spending some time in a school can help you understand if teaching is right for you and give you experience to talk about in your application.
+quote:
```

Comment:
I don't think Zainab did GSE - I think this was more about volunteering independently, so I think would sit better under `Arrange school experience independently`?